src/components/Offices/Finances/index.js

```diff
@@ -1,27 +1,34 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import { officesActions } from '../../../redux/actions';
 import { officesSelectors } from '../../../redux/selectors';
-import styles from './styles.module.scss';
-import Table from "../../Table";
-import { useDispatch, useSelector } from 'react-redux';
+import Table from '../../Table';
 import { formatDollars, formatMerits } from '../../../utils/walletHelpers';
+import { palletIdToAddress } from '../../../utils/pallet';
 
-const ACCOUNTS = [
+const DEFAULT_ACCOUNTS = [
   {
-    name: "Vault",
-    address: "5EYCAe5hvejUE1BUTDSnxDfCqVkADRicSKqbcJrduV1KCDmk",
+    name: 'Vault',
+    address: '5EYCAe5hvejUE1BUTDSnxDfCqVkADRicSKqbcJrduV1KCDmk',
```

Comment:
No hardcoded addresses!
Hardcode palletId instead, and link to the substrate source where it's located.